### PR TITLE
Adding a feature to allow explicitly including github repositories

### PR DESCRIPTION
**Pull request by bushong1**
Updating README with info for REPOS env vars

Let me know if there's any changes you need.

### CODE REVIEW

**Comment by bushong1** [comment]
Working on specs now.

**Comment by bushong1** [comment]
Poke.

**Comment by bushong1** [comment]
@rubenarakelyan @fofr anything needed for this to get merged?  Am i barking up the wrong tree?

**Comment by stig** [comment]
I would love something like this to be supported, as it would stop us having to run two versions of the seal like we currently do.

For some channels we want only PRs from explicitly listed repos (regardless who made the PRs) and therefore have to use the old seal. For other channels we prefer to get all the PRs against any repo made by the team members, and therefore use the current seal version.

**Comment by bushong1** [comment]
Yup, looks like it takes care of that.

**Comment by bushong1** [comment]
@chao-xian Poke

**Comment by bushong1** [comment]
me_irl

File: lib/seal.rb [hunk 1]
```diff
@@ -56,26 +56,29 @@ def team_params(team)
...
       exclude_repos = config['exclude_repos']
+      include_repos = config['include_repos']
       @quotes = config['quotes']
     else
       members = ENV['GITHUB_MEMBERS'] ? ENV['GITHUB_MEMBERS'].split(',') : []
       use_labels = ENV['GITHUB_USE_LABELS'] ? ENV['GITHUB_USE_LABELS'].split(',') : nil
       exclude_labels = ENV['GITHUB_EXCLUDE_LABELS'] ? ENV['GITHUB_EXCLUDE_LABELS'].split(',') : nil
       exclude_titles = ENV['GITHUB_EXCLUDE_TITLES'] ? ENV['GITHUB_EXCLUDE_TITLES'].split(',') : nil
       exclude_repos = ENV['GITHUB_EXCLUDE_REPOS'] ? ENV['GITHUB_EXCLUDE_REPOS'].split(',') : nil
+      exclude_repos = ENV['GITHUB_INCLUDE_REPOS'] ? ENV['GITHUB_INCLUDE_REPOS'].split(',') : nil
```
Comment (rubenarakelyan) [inline lib/seal.rb]: This should be `include_repos`

**Comment by bushong1** [comment]
Fixed, @rubenarakelyan!

**Pull request merged**